views/songlist-db.jade

```diff
@@ -24,7 +24,7 @@ block content
                     td
                         input(type="checkbox", id=song.id)
                     td 
-                        a(href='/song/#{song.id}', font-size="5em") #{song.title}
+                        a(href='/song/#{song.id}?left=#{song.lang._id}&right=', font-size="5em") #{song.title}
```

Comment:
Can we make it so right= becomes optional instead of leaving it hanging like that?